README.md

```diff
@@ -1,4 +1,5 @@
-# FoundryVTT - Forien's Unidentified Items
+# FoundryVTT - Forien's Unidentified Items [MAINTANCE MODE]
```

Comment:
I'm not sure I'd put "Maintenance Mode" - that suggests that new features aren't welcome. The reality is that there isn't a dedicated maintainer at the moment.

Reply:
No problem i removed the annotation